app/views/users/index.html.erb

```diff
@@ -1 +1,12 @@
-<h1>Here's a list of Users</h1>
+<%= stylesheet_link_tag 'users_index' %>
+<section class='users-section'>
+<% @users.each do |user| %>
+    <div class='user-container'>
+        <%= image_tag(user.photo) %>
+        <div class='user-info'>
+            <h2 class='username'><%= user.name %></h2>
+            <p class='number-of-posts'>Number of posts: <%= user.posts_counter %></p>
+        </div>
+    </div>
+<% end %>
```

Comment:
- [ **OPTIONAL** ] I think it would be better if you use `partials` for rails best best practices. As, with a `partial`, you can move the code for rendering a particular piece of a response to its own file instead of repeating the same code every time. I suggest you **[refer this article for more details about partials](https://guides.rubyonrails.org/layouts_and_rendering.html#using-partials)**.
I recommend you to also apply this approach to your other views pages such as `Post`, `Comment`,...